hub/policy/src/integration-test/java/uk/gov/ida/integrationtest/hub/policy/apprule/AuthnRequestFromTransactionResourceIntegrationTest.java

```diff
@@ -55,6 +51,7 @@
 public class AuthnRequestFromTransactionResourceIntegrationTest {
     private static String TEST_SESSION_RESOURCE_PATH = Urls.PolicyUrls.POLICY_ROOT + "test";
     private static final Boolean REGISTERING = TRUE;
+    private static final LevelOfAssurance REQUESTED_LOA = LevelOfAssurance.LEVEL_2;
```

Comment:
I'm not sure that this adds very much. `LevelOfAssurance.LEVEL_2` is already a constant, and the tests would all be a little clearer if we used it directly (rather than hiding it in a class constant)
(The same goes for boolean above to be honest)

Reply:
You could also statically import `LEVEL_2`, which would allow you to need to bother writing `LevelOfAssurance.` throughout these classes

Reply:
I've seen this done in many integration tests - I think it's because of consistency? So the same value is used across the setup and in the test cases. Perhaps a static import would be more succinct.